prow/overlays/smaug/plugins.yaml

```diff
@@ -229,6 +229,30 @@ project_manager:
               org: thoth-station
               state: open
 
+branch_cleaner:
+  preserved_branches:
+    AICoE:
+      - main
+      - master
+    aicoe-aiops:
+      - main
+      - master
+    b4mad:
+      - main
+      - master
+    open-services-group:
+      - main
+      - master
+    operate-first:
+      - main
+      - master
+    os-climate:
+      - main
+      - master
```

Comment:
we should start with only thoth-station repo, as we are using this for the first time.

Reply:
if I understand the [configuration of the plugin](https://github.com/kubernetes/test-infra/blob/e831f6aee73552c33fdc07c5b5ba7dedb29497b4/prow/plugins/plugin-config-documented.yaml#L73-L84) correctly, there is no way to restrict its operation to a specific org... you can configure *branch names* to be preserved org-wide (which is what I did there for `main` and `master`, for all repos on all orgs), but my understanding is that if something is not listed here, then it is not preserved.